gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -1816,13 +1816,17 @@ public List<GbGroup> getSiteSectionsAndGroups() {
 			List<String> viewableGroupIds = this.gradingPermissionService
 					.getViewableGroupsForUser(gradebook.getId(), user.getId(), allGroupIds);
 
+			if (viewableGroupIds == null) {
```

Comment:
isNotEmpy better?

Reply:
It can be null, that's the problem. I was seeing an NPE on the lines below.